codecov.yml

```diff
@@ -0,0 +1,2 @@
+fixes:
+  - "/home/runner/work/rejas/MagicMirror/::"
```

Comment:
This woul dhave to be adjusted to the original repo

Reply:
@rejas I liked the idea of the automatic coverage check and added it to one of my modules. I first tried what you did and I could find online, but couldn't make it work properly.

Then I switched to using the action provided by codecov itself, where I didn't need to specify the path override.

https://github.com/fewieden/MMM-soccer/blob/72f1ef0ba84ece103e639faaadcdfaf1676c0ae6/.github/workflows/coverage.yml#L30-L48

Reply:
Thanks for the hint @fewieden I switched the PR to use the codecov action. Any more helpful hints?